data/json/speech.json

```diff
@@ -1655,12 +1697,42 @@
     "sound": "\"I have done what you asked.  Please let me go!\"",
     "volume": 10
   },
+  {
+    "type": "speech",
+    "speaker": "mon_prototype_cyborg",
+    "sound": "\"I cannot... yet I must. How do you calculate that?\"",
+    "volume": 10
+  },
+  {
+    "type": "speech",
+    "speaker": "mon_prototype_cyborg",
+    "sound": "\"This is a dream. This has to be a dream.\"",
```

Comment:
```suggestion
    "sound": "\"This is a dream.  This has to be a dream.\"",
```

Reply:
I applied the changes! Thanks for catching those!